mir-module/src/main/resources/config/mir/messages_de.properties

```diff
@@ -546,6 +548,7 @@ mir.role.artist                                 = K\u00FCnstler(in):
 mir.role.author                                 = Autor(in):
 mir.role.collector                              = Zusammensteller(in):
 mir.role.contributor                            = Beitragende(r):
+mir.role.correspondingAuthor                    = Korrespondierende(r) Autor(in)
```

Comment:
I think it is not a good style to declare the names together colons. I think the roles were declared with colons at that time because of the author template or <select> instead of a label. The colons should be added there specifically. Basically the names for additional roles are also displayed differently: "Additional roles: <select>".